src/software/simulation/simulator_robot.cpp

```diff
@@ -109,9 +109,7 @@ float SimulatorRobot::getBatteryVoltage()
 {
     // We currently have 4s batteries on the robot that charge up to a little over
     // 16V, so we use 16 here to approximate a fully-charged battery
```

Comment:
Likewise, since we're returning `ROBOT_MAX_BATTERY_VOLTAGE` now and not "using 16 here", I think this comment should be changed to reflect that. You can perhaps move this to the `constants.h` description to explain why it's 16V.